app/views/claims/email_address.html.erb

```diff
@@ -3,19 +3,18 @@
     <%= render("shared/error_summary", instance: current_claim) if current_claim.errors.any? %>
 
     <%= form_for current_claim, url: claim_path do |form| %>
-      <h1 class="govuk-heading-xl">
-        <%= form.label :email_address, t("tslr.questions.email_address")  %>
-      </h1>
+      <%= form_group_tag current_claim do %>
+        <h1 class="govuk-label-wrapper">
+          <%= form.label :email_address, t("tslr.questions.email_address"), class: "govuk-label govuk-label--xl"  %>
+        </h1>
 
-      <span class="govuk-hint">We will only use your email address to update you about your claim.</span>
-
-      <div class="govuk-form-group">
-        <%= form.email_field :email_address, autocomplete: "email", spellcheck: "false", class: "govuk-input" %>
-      </div>
-
-      <div class="govuk-form-group">
-        <%= form.submit "Continue", class: "govuk-button" %>
-      </div>
+        <span class="govuk-hint">We will only use your email address to update you about your claim.</span>
+        <%= errors_tag current_claim, :email_address %>
+        <div class="govuk-form-group">
```

Comment:
Should this be another `form_group_tag`? Or should we lose this div entirely like happened with `full_name`?

Reply:
yep, the second one is redundant!